src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -536,96 +652,41 @@ def create_position_ids_from_inputs_embeds(self, inputs_embeds):
 
         Returns: tf.Tensor
         """
-        seq_length = shape_list(inputs_embeds)[1]
-        position_ids = tf.range(self.padding_idx + 1, seq_length + self.padding_idx + 1, dtype=tf.int32)[tf.newaxis, :]
+        bsz, seq_length = shape_list(tensor=inputs_embeds)[:2]
```

Comment:
`bsz` is a bit too short IMO, `batch_size` should be used (here and two lines below).